### PR TITLE
Added XML comments to the repositoryDeleteConductor methods

### PR DESCRIPTION
**Pull request by SaidShah**
- [x] Related GitHub issue(s) linked in PR description
- [x] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [x] Code builds cleanly (no _additional_ warnings or errors)
- [x] Manually tested
- [x] Automated tests are passing
- [x] No _decreases_ in automated test coverage
- [x] Documentation updated (readme, docs, comments, etc.)
- [x] Localization: No hard-coded error messages in code files (_minimally_ in string constants)

Issue #10

### CODE REVIEW

File: src/AndcultureCode.CSharp.Conductors/RepositoryDeleteConductor.cs [hunk 1]
```diff
@@ -33,14 +44,55 @@ public RepositoryDeleteConductor(IRepository<T> repository)
...
         public virtual IResult<bool> Delete(long id, long? deletedById = default(long?), bool soft = true) => _repository.Delete(id, deletedById, soft);
+
+        /// <summary>
+        /// Ability to delete an entity using the entity itself
+        /// </summary>
+        /// <param name="o">Item to be deleted</param>
+        /// <param name="deletedById">Id of user deleting the item</param>
+        /// <param name="soft">Boolean flag for soft-deleting the item</param>
+        /// <returns></returns>
         public virtual IResult<bool> Delete(T o,     long? deletedById = default(long?), bool soft = true) => _repository.Delete(o,  deletedById, soft);
```
Comment (wintondeshong) [inline src/AndcultureCode.CSharp.Conductors/RepositoryDeleteConductor.cs:72]: Nitpick: Now that everything is broken up by comments, could you remove extra spaces between the parameters? `T o, long? deletedById...`

File: src/AndcultureCode.CSharp.Conductors/RepositoryDeleteConductor.cs [hunk 2]
```diff
@@ -33,14 +44,55 @@ public RepositoryDeleteConductor(IRepository<T> repository)
...
+        /// <param name="soft">Boolean flag for soft-deleting the items</param>
+        /// <returns></returns>
         public virtual IResult<bool> Delete(IEnumerable<T> items ,long? deletedById = default(long?), long batchSize = 100, bool soft = true) => _repository.Delete(items, deletedById, batchSize, soft);
 
+        /// <summary>
+        /// Ability to restore a soft-deleted entity using the entity itself. 
+        /// </summary>
+        /// <param name="o">Entity to be restored</param>
+        /// <returns></returns>
         public virtual IResult<bool> Restore(T o)     => _repository.Restore(o);
```
Comment (wintondeshong) [inline src/AndcultureCode.CSharp.Conductors/RepositoryDeleteConductor.cs:89]: Nitpick: Same as above. Mind removing the space after the signature and the expression body `(T o) =>`

File: src/AndcultureCode.CSharp.Conductors/RepositoryDeleteConductor.cs [hunk 3]
```diff
@@ -33,14 +44,55 @@ public RepositoryDeleteConductor(IRepository<T> repository)
...
+
+        /// <summary>
+        /// Ability to delete a list of entities by batch size.
+        /// </summary>
+        /// <param name="items">List of items to delete</param>
+        /// <param name="deletedById">Id of user deleting the items</param>
+        /// <param name="batchSize">Number of items to include in a batch, defaults to 100</param>
+        /// <param name="soft">Boolean flag for soft-deleting the items</param>
+        /// <returns></returns>
         public virtual IResult<bool> Delete(IEnumerable<T> items ,long? deletedById = default(long?), long batchSize = 100, bool soft = true) => _repository.Delete(items, deletedById, batchSize, soft);
```
Comment (wintondeshong) [inline src/AndcultureCode.CSharp.Conductors/RepositoryDeleteConductor.cs:82]: nitpick: Mind fixing the comma on this line between the 'items' and 'deletedById' params?

**Review by wintondeshong — CHANGES_REQUESTED** [review]
@SaidShah thanks for adding these thorough comments! Just minor adjustments

**Comment by SaidShah** [comment]
@wintondeshong thanks, I took care of those. 👍

**Review by wintondeshong — APPROVED** [review]
Thanks @saidshahdev looks good!

**Pull request merged**